apptools/apptools-android-tests/apptools/check_host.py

```diff
@@ -41,7 +41,7 @@ def test_check_host_without_platforms(self):
         cmd = comm.HOST_PREFIX + comm.PackTools + "crosswalk-app check"
         (return_code, output) = comm.getstatusoutput(cmd)
         self.assertEquals(return_code, 0)
-        self.assertNotIn("ERROR:", output[0])
+        self.assertNotIn("ERROR:", output[0].split('target windows')[0])
```

Comment:
output[0].split('target windows')[0] is the output of testing android host setup. It is used to check  android host setup